docs/UserGuide.adoc

```diff
@@ -132,10 +132,12 @@ From sections 6 to 11, the following notations may be used.
 |Notation |Meaning
 
 |[ ] |Necessary field
-|< > |Optional field. e.g. `n/NAME <g/GOAL>` can be used as `n/Spicy Chicken g/Bulk like the Hulk` or as `n/Spicy Chicken`
+|< > |Optional field. e.g. `n/NAME <s/STEP>` can be used as `n/Spicy Chicken s/Step 1` or just `n/Spicy Chicken`
```

Comment:
I think we should put the 'NAME' and 'STEP' in lower case letters to follow the same format we used in the UG later